src/Providers/DataGrid/Wijmo/Features/ColumnPicker.ts

```diff
@@ -168,6 +169,39 @@ namespace Providers.DataGrid.Wijmo.Feature {
 			);
 		}
 
+		// This code fixes WJM-34234
+		private _issueWorkaround(e) {
+			if (e.ctrlKey && e.code === 'KeyA') {
+				const checkedItems = [...this._theColumnPicker.checkedItems];
+				this._theColumnPicker._children.forEach((item, index) => {
+					const input = item.querySelector('input');
+
+					// If the checkbox item is disabled
+					if ((input && input.disabled) || item.classList.contains('wj-state-disabled')) {
```

Comment:
Do we need to compare with `input && input.disabled`, as in, is it not enough to use the second part of the condition (`item.classList.contains('wj-state-disabled')`)?

Reply:
I understand your point, but this is what we have in the workaround provided, so I just assumed that it should be like this.

Reply:
Got it, thank you!